### DISCUSSION TITLE
acb544c/

### DISCUSSION THREAD

**Discussion by giscus** [james1BadCreeper/james1BadCreeper.github.io#14] [Announcements]
# acb544c/

本文为 james1 CSP-S 2022 的记录（同时把 NOIP 一起写掉了），大家快来看这个水浸蒟蒻是怎么爆零的吧！
 - james1BadCreeper - james1 的博客

https://james1badcreeper.github.io/acb544c/

**Comment by Zimo-666** [comment]
明明今天冬至，心还在那盛夏。感谢你的陪伴。
